extensions-core/hdfs-storage/src/test/java/org/apache/druid/inputsource/hdfs/HdfsInputSourceTest.java

```diff
@@ -66,6 +67,10 @@
 @RunWith(Enclosed.class)
 public class HdfsInputSourceTest
 {
+  static {
+    NullHandling.initializeForTests();
```

Comment:
nit: you can extend `InitializedNullHandlingTest` instead of adding a static block.

Reply:
sure, I can do that, I merely copied this from other existing tests

Reply:
Thanks. I think they were added before we add `InitializeNullHandlingTest`. We could clean them up someday.

Reply:
updated